src/Command/CheckRequirements.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Command;
+
+use Manala\Config\Requirement\Common\RequirementLevel;
+use Manala\Config\Requirement\Factory\HandlerFactoryResolver;
+use Manala\Config\Requirement\Requirement;
+use Manala\Config\Requirement\RequirementChecker;
+use Manala\Config\Requirement\Violation\RequirementViolation;
```

Comment:
Am i the only one prefering to have a single "Manala\Config\Requirement\Violation" use, and some "Violation\RequirementViolation" or "Violation\RequirementViolationLabelBuilder" inside code ?